lib/core/src/lib/form/components/widgets/core/form-field.model.spec.ts

```diff
@@ -277,7 +305,8 @@ describe('FormFieldModel', () => {
         const currentDate = new Date();
         const expectedDateFormat = `${DateFnsUtils.formatDate(currentDate, 'yyyy-MM-dd')}T00:00:00.000Z`;
 
-        expect(field.value).toEqual(currentDate);
+        expect(field.value.getDate()).toEqual(currentDate.getDate());
+        expect(field.value.getHours()).toEqual(currentDate.getHours());
```

Comment:
It is the thing mentioned by Tom [here](https://github.com/Alfresco/alfresco-ng2-components/pull/9939/files) (flaky unit test)

Reply:
Needed to pass units (not related with actual issue)